lib/market.rb

```diff
@@ -0,0 +1,23 @@
+class Market
+  attr_reader :name, :vendors
+
+  def initialize(name)
+    @name = name
+    @vendors = []
+  end
+
+  def add_vendor(vendor)
+    @vendors << vendor
+  end
+
+  def vendor_names
+    @vendors.map{|vendor| vendor.name }
+  end
+
+  def vendors_that_sell(item)
```

Comment:
You did a nice job describing this method in your video - I particularly liked that you covered both the test and the method itself!